content/rust/concepts/functions/functions.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Functions'
+Description: 'Functions are the block of reusable code that can perform similar related actions.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Game Development'
+Tags:
+  - 'Functions'
+CatalogContent:
+  - 'Computer Science'
+  - 'rust-for-programmers'
+---
+
+**Rust functions** are declared using the `fn` keyword. Its arguments are type annotated, just like variables, and, if the function returns a value, the return type must be specified after an arrow `->`.
+Rust code uses _snake_case_ as the conventional style for function and variable names, in which all letters are lowercase and underscores separate words.
+
```

Comment:
Generally, there should be a couple of sections all separated by a header that will highlight a particular aspect of functions with this language, such as return values, best practices, etc